index.html

```diff
@@ -11,5 +11,38 @@
     <title>My Portfolio2</title>
 </head>
 <body>
+  <div class="gen-container">
+    <nav>
+      <h3 class="logo">
+        My Logo
+      </h3>
+      <div class="menu-bars">
+        <img src="header-section\bars.svg" alt="">
+      </div>
```

Comment:
- [ ] [OPTIONAL] it would be better if you remove the white space at the top in order to match the Figma design 

<img width="382" alt="Screen Shot 2022-09-26 at 17 43 19" src="https://user-images.githubusercontent.com/61790587/192335136-b9c5ad86-9a24-475b-8aa8-e31ddc25779c.png">